packages/contracts/test/contracts/OVM/execution/OVM_ExecutionManager/nuisance-gas.spec.ts

```diff
@@ -188,7 +188,7 @@ const test_nuisanceGas: TestDefinition = {
             // This is because there is natural gas consumption between the ovmCALL(GAS/2) and ovmCREATE, which allots nuisance gas via _getNuisanceGasLimit.
             // This means that the ovmCREATE exception, DOES consumes all nuisance gas allotted, but that allotment
             // is less than the full OVM_TX_GAS_LIMIT / 2 which is alloted to the parent ovmCALL.
-            nuisanceGasLeft: 4184829,
+            nuisanceGasLeft: 4197042,
```

Comment:
The current master value is the correct one, this should be reverted.

I had changed this [here](https://github.com/ethereum-optimism/optimism-monorepo-old/pull/52/commits/65387a214c8a4062f688c158ebc7979ef197c33b) because there was some bug in smock that was fixed [here](https://github.com/ethereum-optimism/smock/commit/96e7535c81f30a4e66f08335473da25ae35ffa4b).

@smartcontracts has some context on this, but the post-state checks were broken because of the issue mentioned in smock

Reply:
Ah okay, @smartcontracts told me that 4197042 was the correct one.